examples/shapewin.py

```diff
@@ -165,7 +165,7 @@ def loop(self):
                                            self.sub_pm)
 
             # Shape has changed
-            elif e.type == self.d.extension_event.ShapeNotify:
+            elif e.type == shape.Event.Notify:
```

Comment:
How does this pick up the display-specific event ID for the extension event?  Looks to me like shape.Event.Notify will just be 0.